src/config.rs

```diff
@@ -252,13 +250,75 @@ fn get_global_config_path() -> Option<PathBuf> {
     Some(PathBuf::from("/etc/rustic"))
 }
 
+#[derive(Debug, Default, Clone, Serialize, Deserialize, Merge)]
+#[serde(default, rename_all = "kebab-case", deny_unknown_fields)]
+pub struct Hooks {
+    /// Call this command before every rustic operation
+    pub run_before: CommandInput,
+
+    /// Call this command after every successful rustic operation
+    pub run_after: CommandInput,
+
+    /// Call this command after every rustic operation
+    pub run_failed: CommandInput,
```

Comment:
typo in comment: "every" → "every failed"

Reply:
Good catch, thanks! Is fixed now.